meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
+DIR=/usr/share/www/img
+if [ "${model[1]}" = '"S0"' ]  || [ "${model[1]}" = '"D0"' ]; then
+    echo "Using IBM logo"
+elif [ "${model[1]}" = '"J0"' ]; then
+    echo "loading IPS logo"
+    mount --bind ${DIR}/inspur-login-logo.svg.gz ${DIR}/login-company-logo.svg.gz
+    mount --bind ${DIR}/inspur-logo-header.svg.gz ${DIR}/logo-header.svg.gz
+    mount --bind ${DIR}/blankLogo.svg.gz  /usr/share/www/bee-2-light.svg.gz
```

Comment:
If you have the browser tools open, will still see a call to GET https://${bmc}/bee-2-light.svg which is weird, but we are out of time, so can fix for 1030.